MAUI/AIAssistView/data-binding.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Data Binding in .NET MAUI AI Assist View Control | Syncfusion
+description: Learn here all about Data Binding support in Syncfusion .NET MAUI AI Assist View (SfAIAssistView) control, its elements, and more.
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Data Binding in .NET MAUI AI Assist View (SfAIAssistView)
+
+## Generate the assist request & response items
+
+To add an `AssistItem` to the `ViewModel.AssistItems` collection with specific values for profile details, text, and the `IsRequest` property, follow the steps below:
+
+1. Start by creating an instance of the `AssistItem` class. This item will represent either a user request or a response received from the AI service.
+
+2. The Assistitem has the following members, which provides information for the request/response items,
+
+* `Profile`: Provides information for the user details.
+* `Text`: Describes the text content of the assist item (e.g., the request text from the user or the response text from the AI).
+* `IsRequested`: When the assist item represents a user request, set the `IsRequest` property to `True`. If it's a response item from the AI service, set `IsRequest` to `False`.
+* `DateTime`: To display item created or received time.
+* `RequestItem`: Describes the data associated with the request item.
```

Comment:
Used to hold data (request item) associated with response item. Default value is `null`.

Reply:
Updated